static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx

```diff
@@ -429,9 +433,24 @@ function getSpanEvidenceValue(span: Span | null): string {
     return span.op;
   }
 
+  if (span.op === 'db' && span.description) {
+    return (
+      <StyledCodeSnippet language="sql">
+        {`${span.op} - ${formatter.toString(span.description)}`}
```

Comment:
Out of curiosity, why add the `db -` prefix here?

Reply:
that's what we did before so I'm matching the old logic. I guess it's done to be more explicit which span op it was?

Reply:
Up to you, though I think it might be weird to have as it part of the code snippet! e.g., if someone clicks "Copy", the `db -` will be part of it

Reply:
removed the `db` prefix since it's silly to have it there